wurst/lib/ItemRecipes.wurst

```diff
@@ -0,0 +1,1431 @@
+            ..addItem(ITEM_ELK_SKIN_GLOVES)
+            ..addItem(ITEM_WOLF_SKIN_GLOVES)
+            ..addItem(ITEM_BEAR_SKIN_GLOVES)
+        ..endBatch()
+        ..addItem(ITEM_STEEL_INGOT)
+        ..addItem(ITEM_STEEL_INGOT)
+    recipe.requirement = requirement
+
+    recipe = new ItemRecipe(ITEM_STEEL_COAT, 0, true, true, true)
+        ..startBatch()
+            ..addItem(ITEM_ELK_SKIN_COAT)
+            ..addItem(ITEM_WOLF_SKIN_COAT)
+            ..addItem(ITEM_BEAR_SKIN_COAT)
+        ..endBatch()
+        ..addItem(ITEM_STEEL_INGOT)
+        ..addItem(ITEM_STEEL_INGOT)
+    recipe.requirement = requirement
+
+    recipe = new ItemRecipe(ITEM_IRON_INGOT, 0, true, true, true)
+        ..addItem(ITEM_STONE)
+        ..addItem(ITEM_FLINT)
+    recipe.requirement = requirement
+
+    recipe = new ItemRecipe(ITEM_STEEL_INGOT, 0, true, true, true)
+        ..addItem(ITEM_IRON_INGOT)
+        ..addItem(ITEM_IRON_INGOT)
+        ..addItem(ITEM_FLINT)
+        ..addItem(ITEM_FLINT)
+    recipe.requirement = requirement
+
+    recipe = new ItemRecipe(ITEM_SPEAR, 1, true, false, true)
+        ..addItem(ITEM_STONE)
+        ..startBatch()
+            ..addItem(ITEM_BONE)
+            ..addItem(ITEM_STICK)
+        ..endBatch()
+    recipe.requirement = requirement
+
+    recipe = new ItemRecipe(ITEM_FLINT_AXE, 0, true, false, true)
+        ..startBatch()
+            ..addItem(ITEM_BONE)
+            ..addItem(ITEM_STICK)
+        ..endBatch()
+        ..addItem(ITEM_FLINT)
+        ..addItem(ITEM_FLINT)
+    recipe.requirement = requirement
+
+    recipe = new ItemRecipe(ITEM_STONE_AXE, 0, true, false, true)
+        ..startBatch()
+            ..addItem(ITEM_BONE)
+            ..addItem(ITEM_STICK)
+        ..endBatch()
+        ..addItem(ITEM_STONE)
+        ..addItem(ITEM_STONE)
+    recipe.requirement = requirement
+
+    recipe = new ItemRecipe(ITEM_IRON_AXE, 0, true, false, true)
+        ..startBatch()
+            ..addItem(ITEM_BONE)
+            ..addItem(ITEM_STICK)
+        ..endBatch()
+        ..addItem(ITEM_IRON_INGOT)
+        ..addItem(ITEM_IRON_INGOT)
+    recipe.requirement = requirement
+
+    recipe = new ItemRecipe(ITEM_STEEL_AXE, 0, true, false, true)
+        ..startBatch()
+            ..addItem(ITEM_BONE)
+            ..addItem(ITEM_STICK)
+        ..endBatch()
+        ..addItem(ITEM_STEEL_INGOT)
+        ..addItem(ITEM_STEEL_INGOT)
+    recipe.requirement = requirement
+
+    recipe = new ItemRecipe(ITEM_LOADED_BONE, 1, true, true, true)
+        ..addItem(ITEM_BLOW_GUN)
+        ..addItem(ITEM_BONE)
+    recipe.requirement = requirement
+
+    recipe = new ItemRecipe(ITEM_LOADED_THISTLES, 15, true, true, true)
+        ..addItem(ITEM_BLOW_GUN)
+        ..addItem(ITEM_THISTLES)
+    recipe.requirement = requirement
+        // add requirement for troll unit to
+
+    recipe = new ItemRecipe(ITEM_LOADED_DARK_THISTLES, 12, true, true, true)
+        ..addItem(ITEM_BLOW_GUN)
+        ..addItem(ITEM_DARK_THISTLES)
```

Comment:
What happens with the other three?

Reply:
Is this sensitive to the number of charges that are in the `ITEM_DARK_THISTLES`?